src/manifest/dbtProject.ts

```diff
@@ -695,21 +696,40 @@ export class DBTProject implements Disposable {
   }
 
   async getColumnValues(model: string, column: string) {
-    this.terminal.debug(
-      "getColumnValues",
-      "finding distinct values for column",
-      true,
-      { model, column },
-    );
-    const query = `select ${column} from {{ ref('${model}')}} group by ${column}`;
-    const queryExecution = await this.dbtProjectIntegration.executeSQL(
-      query,
-      100, // setting this 100 as executeSql needs a limit and distinct values will be usually less in number
-      model,
+    this.telemetry.startTelemetryEvent(
```

Comment:
`startTelemetryEvent` will start tracking the time taken for completing an event